Week1/homework/exercise8.js

```diff
@@ -0,0 +1,60 @@
+var item1 = ("On wednesday we wear pink.");
+var item2 = ("Except I am wearing red...");
+var item3 = {HeyHo};
+var item4 = {LetsGo};
+
+if (typeof item1 == typeof item2)
```

Comment:
Try to make it a habit of using triple equals instead of double equals when doing comparison in JavaScript. 🙂